recipes/yaehmop/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "yaehmop" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/greglandrum/yaehmop/archive/master.tar.gz
+  sha256: d1d69e0d46bf753265378768ebd72752f8a61844395390e4c623a9beb0d58525
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+
+test:
+  commands:
+    - yaehmop -v
+
+about:
+  home: https://github.com/greglandrum/yaehmop
+  license: BSD-2-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
```

Comment:
In the downloaded source, the license is called `license.txt` (notice it is lowercase). Change this line to lowercase so conda pulls the license from the download.